README.md

```diff
@@ -133,3 +134,13 @@ SYNC Updating Policy: Test policy 1
 
 The command provides output to identify which actions have been taken. If using a Tyk Gateway, the Gateway will be
 automatically hot-reloaded.
+
+
+## Example: Getting the current Tyk Sync version
+
+In order to get the current Tyk Sync version, we need to run the version command.
```

Comment:
Please change this to say:
`To check the current Tyk Sync version, we need to run the version command:`